tests/ui/codegen/overflow-during-mono.rs

```diff
@@ -1,5 +1,4 @@
 //@ build-fail
-//~^ ERROR overflow evaluating the requirement
 
```

Comment:
```suggestion
//~^ ERROR reached the type-length limit while instantiating
```
so that we don't accidentally go back to getting overflow here

Reply:
Can't do that -- it's not pointing to this crate. Added an `error-pattern` instead.